CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [Unreleased]
+
+### Added
+
+- Added `--payload_size` parameter to configure the maximum payload size for
```

Comment:
`payload_size` is not a clear name:
since this is a firecracker-level parameter, it needs a more descriptive name as to what it relates.

For example: `--http_api_max_payload_size`